core/modules/layout/includes/layout.layout.inc

```diff
@@ -76,6 +76,21 @@ function layout_layout_context_info() {
   return $info;
 }
 
+/**
+ * Implements hook_layout_relationship_info().
+ */
+function layout_layout_relationship_info() {
+  $info['author_from_node'] = array(
+    'title' => t('Author from content'),
+    'class' => 'LayoutRelationshipAuthorFromNode',
+    'context' => 'node',
+    'context_label' => t('Node'),
```

Comment:
Shouldn't this also be `Content`?

Reply:
Existing Node contexts still show "Node", not "Content", so this is consistent with what we have at the moment.